core/tests/ts-integration/src/context-owner.ts

```diff
@@ -624,6 +625,9 @@ export class TestContextOwner {
             // Then propagate the exception.
             throw error;
         }
+        if (this.env.l2NodePid !== undefined) {
```

Comment:
maybe blend it with others?
```
if (this.env.l2NodePid !== undefined) {
    this.reporter.startAction(`Terminating L2 node process`);
    await killPidWithAllChilds(this.env.l2NodePid, 9);
    this.reporter.finishAction();
}
```
is it bad to use l2Node here? then we could do `l2Node.killAndWaitForShutdown()`. like if tests are running one by one?

Reply:
it's hard to use complex structures in env, since we do json serialization/deserialization there